### PR TITLE
Add support for Wheel Distribution

### PR DESCRIPTION
**Pull request by vbaddi**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -40,7 +40,7 @@ docs = ["Sphinx==7.1.2","sphinx-rtd-theme==2.0.0","myst-parser==3.0.1"]
 quality = ["black", "ruff", "hf_doc_builder@git+https://github.com/huggingface/doc-builder.git"]
 
 [build-system]
-requires = ["setuptools>=62.0.0"]
+requires = ["setuptools>=62.0.0", "wheel", "build"]
```
Comment (irajagop) [inline pyproject.toml:43]: These packages need not be specified here. They will not be installed automatically before running build. They need to be installed manually.

File: README.md [hunk 2]
```diff
@@ -53,6 +53,10 @@ pip install -U pip
 # Clone and Install the QEfficient Repo.
 pip install git+https://github.com/quic/efficient-transformers
 
+# Or build wheel package using the below command.
+python -m build --wheel --outdir dist
+pip install dist/QEfficient-0.0.1.dev0-py3-none-any.whl
```
Comment (irajagop) [inline README.md:58]: We can specify the installation of `build` and `wheel` packages here.

**Review by irajagop — CHANGES_REQUESTED** [review]

File: pyproject.toml (hunk 1 above)
Comment (vbaddi) [reply pyproject.toml]: Agree, removing them from here.

File: README.md (hunk 2 above)
Comment (vbaddi) [reply README.md:59]: installation updated here.

**Review by irajagop — APPROVED** [review]

**Review by ochougul — APPROVED** [review]

**Pull request merged**